src/ICS26Router.sol

```diff
@@ -125,9 +124,8 @@ contract ICS26Router is IICS26Router, IBCStore, Ownable, IICS26RouterErrors, Ree
         });
 
         ics02Client.getClient(msg_.packet.destChannel).membership(membershipMsg);
-        uint64 nanoTimestamp = uint64(block.timestamp * 1_000_000_000);
-        if (msg_.packet.timeoutTimestamp <= nanoTimestamp) {
-            revert IBCInvalidTimeoutTimestamp(msg_.packet.timeoutTimestamp, nanoTimestamp);
+        if (msg_.packet.timeoutTimestamp <= block.timestamp) {
```

Comment:
The implications of this, combined with the packetCommitment creation that changes from seconds to nanoseconds, as far as I can tell are:
* The packet from the other chain will need to use seconds
* The packet commitment from the other chain will need to also commit to nanoseconds

Reply:
The packet commitment from the other chain is ibc-go, so it already uses nanoseconds.

We do not impose any requirements on the packet being submitted to the counterparty chain. We only require that packets submitted to this chain by the relayer use seconds.

Reply:
Ok, I see. All packet and light clients need to be nanoseconds on ibc-go and all packets and light clients need to be second in the evm implementation. And the relayers just need to know about translate the timestamps, is that correct?